docs/src/config.md

```diff
@@ -235,10 +235,13 @@ required. A `reducer` can be specified to apply to the model output, see for mor
 information the following section [Output CSV section](@ref). When a `map` is provided to
 extract data for certain locations (e.g. `gauges`) or areas (e.g. `subcatchment`), the
 NetCDF location names are extracted from these maps. For a specific location (grid cell) a
-`location` is required. In the section [Output CSV section](@ref), similar functionality is
-available for CSV. For integration with Delft-FEWS, see also [Run from Delft-FEWS](@ref),
-it is recommended to write scalar data to NetCDF format since the General Adapter of
-Delft-FEWS can ingest this data format directly.
+`location` is required. Model parameters with the extra dimension `layer` for layered model
```

Comment:
small thing, but I suggest to write:
"Model parameters **and variables** with the extra dimension ..."

Reply:
Thanks for the suggestion, this has been changed in commit https://github.com/Deltares/Wflow.jl/pull/193/commits/089858161c6397516f5d69cf31bf90e8ade8253f.